.npmignore

```diff
@@ -1,20 +1,8 @@
-# Configuration files
+# Development-specific files
 .deploy.env
 .editorconfig
 .eslintrc.js
 .gitattributes
 .gitignore
 .travis.yml
-
-# Tests
-test-utils/
-tests/
-
-# Documentation
-docs/
-examples/
 CONTRIBUTING.md
-
-# Browser stub (use index.js w/ a bundler or mithril.js w/o one instead)
-module/
```

Comment:
I'd like to use the module stub as a dependency in another lib, keeping it in the bundle is better for me (I could point NPM to an old git commit/prerelease version, but I'd rather not do that).